### ISSUE TITLE
Overlaying banner disturb the interaction of the runner

### ISSUE THREAD

**Issue by sbathke** [automate-website/jwebrobot#54]
In the case when a bottom-fixed aligned banner is displayed and the user is choose to click (or move and then click) on an element which is out of the viewport, then the banner will be clicked instead.
Its possible that the click command is moving to the element which is still hidden by the banner and then the click command is executed (see screenshots).

With banner:
![image](https://cloud.githubusercontent.com/assets/11196644/22066794/e4c90962-dd8e-11e6-8260-baf2f1e0357c.png)

and without:
![image](https://cloud.githubusercontent.com/assets/11196644/22070845/af66764c-dd9d-11e6-95b5-c71a283b82ce.png)

**Comment by dimw** [comment]
@sbathke Indeed, I have already observed similar issue where an absolutely positioned element covered a control elements. It has to be discussed how to deal with such designs. A real user would probably scroll down until the buttons become visible.

A (temporary) workaround could the placement of a dedicated [`move` step](https://github.com/automate-website/waml#move) and move the viewport to a certain auxiliary point where the buttons are clickable.
